src/main.ts

```diff
@@ -89,8 +89,10 @@ async function run(): Promise<void> {
     } else {
       core.info('No URL set. Only HttpsTrigger Cloud Functions have URL.');
     }
-  } catch (error) {
-    core.setFailed(error.message);
+  } catch (err) {
```

Comment:
Apparently this is no longer a concrete type in `actions/core` 1.6+. Either way, I think it's good to prefix these errors with a well-known string that clearly identifies the name of the action that caught the exception.

Reply:
Yup sgtm